core/templates/dev/head/pages/skill_player/concept_card_directive.html

```diff
@@ -0,0 +1,78 @@
+<div class="conversation-skin-tutor-card">
+  <div class="conversation-skin-tutor-card-content">
+    <div class="conversation-skin-tutor-card-top-section">
+      <img class="conversation-skin-oppia-avatar"
+           ng-src="<[OPPIA_AVATAR_IMAGE_URL]>" alt="">
+      <div class="rte-viewer conversation-skin-tutor-card-top-content">
+        <angular-html-bind html-data="skillExplanation">
+        </angular-html-bind>
+        <div ng-if="workedExamples.length > 0" style="margin-top: 20px;">
+          <h3> Worked Examples: </h3>
+          <p ng-repeat="workedExample in workedExamples track by $index"
+             class="worked-example"
+             ng-if="$index <= maximumVisibleExampleIndex">
+            <label> <[$index + 1]>: &nbsp;</label>
+            <angular-html-bind html-data="workedExample">
+            </angular-html-bind>
+          </p>
+          <button class="btn btn-primary add-worked-example"
+                  ng-click="viewNextExample()"
+                  ng-if="maximumVisibleExampleIndex < (workedExamples.length - 1)">
+            <i class="material-icons"> add </i> Worked Example
```

Comment:
Btw, we should be i18n'ing the fields for skills, or at least giving them audio subtitles, similar to explorations. Do we?

Reply:
Done, added translate for Worked Example. What should be done about audio translation?

Reply:
Separate PR, probably. Might want to consult with @DubeySandeep. But the main thing for now is to ensure that the backend models are correct and support translations. We can add the additional editing functionality later.

Reply:
Ok.